pifpaf/__main__.py

```diff
@@ -185,7 +185,7 @@ def expand_urls_var(url):
 
             putenv("PID", str(os.getpid()))
             putenv("DAEMON", daemon)
-            url = os.getenv(driver.env_prefix + "_URL")
+            url = os.getenv(driver.env_prefix + "_URL", "")
```

Comment:
This seems unrelated to this PR. If it's needed, can you put that into another PR?

Reply:
will do, it was crashing cause the new driver didn't put an env variable prefixed with _URL.